pkg/controller/restic_repository_controller.go

```diff
@@ -18,184 +18,155 @@ package controller
 
 import (
 	"context"
-	"encoding/json"
 	"strings"
 	"time"
 
-	jsonpatch "github.com/evanphx/json-patch"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/clock"
-	"k8s.io/client-go/tools/cache"
+
+	"sigs.k8s.io/cluster-api/util/patch"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
-	velerov1client "github.com/vmware-tanzu/velero/pkg/generated/clientset/versioned/typed/velero/v1"
-	velerov1informers "github.com/vmware-tanzu/velero/pkg/generated/informers/externalversions/velero/v1"
-	velerov1listers "github.com/vmware-tanzu/velero/pkg/generated/listers/velero/v1"
 	"github.com/vmware-tanzu/velero/pkg/restic"
-
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	"github.com/vmware-tanzu/velero/pkg/util/kube"
 )
 
-type resticRepositoryController struct {
-	*genericController
+const (
+	repoSyncPeriod = 5 * time.Minute
+)
 
-	resticRepositoryClient      velerov1client.ResticRepositoriesGetter
-	resticRepositoryLister      velerov1listers.ResticRepositoryLister
-	kbClient                    client.Client
-	repositoryManager           restic.RepositoryManager
+type resticRepoConciler struct {
```

Comment:
This is trivial but for consistency `Conciler` -> `Reconciler`?

Reply:
yeah, I'll change it